airbyte-cdk/python/unit_tests/test_connector.py

```diff
@@ -71,3 +80,53 @@ def test_write_config(integration, mock_config):
     integration.write_config(mock_config, str(config_path))
     with open(config_path, "r") as actual:
         assert mock_config == json.loads(actual.read())
+
+
+class TestConnectorSpec:
```

Comment:
I think these unit tests should fit under the `TestAirbyteSpec` class since we're testing the `spec` method from the `AirbyteSpec` class in `connector.py`

Reply:
I didn't put these in there because `AirbyteSpec` is actually a different, unrelated class. Here we're testing the `spec` method of the `Connector` class, defined in the same file. I was looking around and actually could not find `AirbyteSpec` being used anywhere other than tests, so my guess is this was being used in a previous version of the CDK but is not anymore.

Reply:
Gotcha, I misread `connector.py` as I was reviewing, my bad!